merkle/compact_merkle_tree_test.go

```diff
@@ -92,8 +92,8 @@ func TestAddingLeaves(t *testing.T) {
 			if got, want := tree.CurrentRoot(), roots[i]; !bytes.Equal(got, want) {
 				t.Errorf("CurrentRoot()=%v, want %v", got, want)
 			}
-			if got, want := tree.Hashes(), hasheses[i]; !reflect.DeepEqual(got, want) {
-				t.Errorf("Hashes()=%v, want %v", got, want)
+			if diff := pretty.Compare(tree.Hashes(), hashes[i]); diff != "" {
+				t.Errorf("post-Hashes() diff:\n%v", diff)
```

Comment:
Sorry I must have been especially unclear. There are several "post-Hashes() diff:\n%v" messages in the same test so you don't know which tree triggered it. Same for some others.

It was pre-existing so I'm not too hung up on it being fixed in this PR.